results/docs/Control_benchmarks_analysis.md

```diff
@@ -0,0 +1,37 @@
+# Control Benchmarks 
+
+The suite of control conditional benchmarks which is a part of the [university of wisconsin micro-benchmarks suite](https://github.com/VerticalResearchGroup/microbench) can be used to try and understand the difference in performance between the intel skylake processor and the branch prediction unit in gem5.
+
+Control Conditional Benchmarks:
+
+- CCa : completely baised branches
+
+- CCe : easy to predict -- branch pattern (10101010...)
```

Comment:
```suggestion
- CCe : easy to predict -- branch pattern (i.e., 10101010...)
```

Is it exactly 1010101 or "like" 1010101 (should that be e.g., or i.e.,?)

Reply:
@tv-reddy I am not sure, but I think it is the exact pattern rite?